lib/pages/events_page.dart

```diff
@@ -38,6 +46,22 @@ class _EventsPageState extends State<EventsPage> {
       }
     }
 
+    final BottomNavigationBar bottomNavigationBar = new BottomNavigationBar(
+      items: const <BottomNavigationBarItem>[
+        BottomNavigationBarItem(
+          icon: Icon(Icons.home),
+          title: Text('All'),
+        ),
+        BottomNavigationBarItem(
+          icon: Icon(Icons.calendar_today),
+          title: Text('Today'),
+        ),
+      ],
+      currentIndex: _selectedIndex,
+      selectedItemColor: Colors.red[800],
```

Comment:
Probably better to grab a color from the theme similar to `Theme.of(context).buttonColor`